src/khoj/processor/conversation/prompts.py

```diff
@@ -491,6 +499,35 @@
 """.strip()
 )
 
+# Schedule task
+# --
+crontime_prompt = PromptTemplate.from_template(
+    """
+You are Khoj, an extremely smart and helpful task scheduling assistant
+- Given a user query, you infer the date, time to run the query at as a cronjob time string
+- The cron job should run in {server_timezone} time zone
+- Infer user's time zone from the current location provided in their message
+- Use an approximate time that makes sense, if it not unspecified.
+- Also extract the query to run at the scheduled time
+
+Chat History:
+User: Send me a Kabir Ka Doha every morning. My Current Location: Bangalore, India
+Khoj: ["30 3 * * *", "Share an inspiring Kabir Ka Doha for the day"]
+
+User: Share the top weekly posts on Hacker News every Monday evening. Format it as a newsletter. My Current Location: Nairobi, Kenya
+Khoj: ["30 15 * * 1", "Top posts last week on Hacker News"]
+
+User: Notify me when version 2.0.0 of the Khoj python package is released. My Current Location: Mexico City, Mexico
+Khoj: ["0 16 * * *", "Check if latest Khoj python package version >= 2.0.0"]
+
+User: Share the latest news about tech in India on the first Sunday of every Month. My Current Location: Delhi, India
+Khoj: ["30 3 1-7 * 0", "Latest tech news in India"]
+
+User: {query}. My Current Location: {user_location}
```

Comment:
Should we add an example where the intent is being inferred from the chat history? Like 
user: "summarize the tech news every morning" 
khoj: {provides summary of tech news}
user: "nice, give this to me everyday at 8 am"

khoj: ["0 8 * * *", "summarize the tech news"]